src/System Application/App/File System/src/Scenario/FileAccountScenario.Table.al

```diff
@@ -0,0 +1,65 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+/// <summary>
+/// Temporary table used to display the tree structure in "File Scenario Setup".
+/// </summary>
+table 9453 "File Account Scenario"
+{
+    Access = Internal;
+    InherentPermissions = X;
+    InherentEntitlements = X;
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; Scenario; Integer)
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(2; Connector; Enum "File System Connector")
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(3; "Account Id"; Guid)
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(4; "Display Name"; Text[2048])
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(5; Default; Boolean)
+        {
+            DataClassification = SystemMetadata;
+        }
+        field(6; EntryType; Option)
```

Comment:
This is used as parameters where there is `Option` datatype without the options listed, perhaps Enum would be the better choice